app/src/views/private/private-view/components/private-view.vue

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+export interface PrivateViewProps {
+	/** Title to show in the header bar. Can be overriden through the #title slot */
+	title?: string;
+
+	/** Render an icon before the title */
+	icon?: string;
+
+	/** What color to render the icon in */
+	iconColor?: string;
+
+	/** Render a history back button in place of the title prepend icon */
+	showBack?: boolean;
+}
+</script>
+
+<script setup lang="ts">
+import { useSettingsStore } from '@/stores/settings';
+import { useUserStore } from '@/stores/user';
+import { useCookies } from '@vueuse/integrations/useCookies';
+import { computed } from 'vue';
+import NotificationDialogs from '../../components/notification-dialogs.vue';
+import NotificationsDrawer from '../../components/notifications-drawer.vue';
+import PrivateViewNoAppAccess from './private-view-no-app-access.vue';
+import PrivateViewRoot from './private-view-root.vue';
+
+const userStore = useUserStore();
+
+const appAccess = computed(() => {
+	if (!userStore.currentUser) return true;
+	return userStore.currentUser?.app_access || false;
+});
+
+defineProps<PrivateViewProps>();
```

Comment:
Let’s move `defineProps` to the top, below the imports, to improve file structure and readability.

```suggestion
defineProps<PrivateViewProps>();

const userStore = useUserStore();

const appAccess = computed(() => {
	if (!userStore.currentUser) return true;
	return userStore.currentUser?.app_access || false;
});
```